problem_builder/tests/integration/test_questionnaire.py

```diff
@@ -192,15 +193,10 @@ def test_mrq_feedback_popups(self):
         ]
         self.popup_check(mentoring, item_feedbacks, prefix='div[data-name="mrq_1_1_7"]')
 
-        for index, _ in enumerate(item_feedbacks):
-            choice_wrapper = mentoring.find_elements_by_css_selector('div[data-name="mrq_1_1_7"]' + " .choice")[index]
-            choice_wrapper.find_element_by_css_selector(".choice-selector input").click()
-            item_feedback_icon = choice_wrapper.find_element_by_css_selector(".choice-result")
-            item_feedback_icon.click()  # clicking on item feedback icon
-            item_feedback_popup = choice_wrapper.find_element_by_css_selector(".choice-tips")
-
-            self.assertFalse(item_feedback_popup.is_displayed())
-            self.assertEqual(item_feedback_popup.text, "")
```

Comment:
Why are these tests being removed when the UI elements have not been removed?

Reply:
Hi @xitij2000, thanks for reviewing. As per the changes on this PR #256, these scenarios become obsolete. so I tried to update accordingly. Actually, I had updated this test case in the past as we are facing empty pop up without performing submission after rendering a page. By the recent changes of #256, we wouldn't face these scenarios.

Reply:
I don't see how these are obsolete. The span is now a button in a number of places, but the actual functionality being tested here is intact. This is existing tests instead of adapting them.

Reply:
Yeah, you are right on it's changed into a button but I'm saying in the context of CSS changes that are applied to this button as it will not be displayed until it is selected during the form submission. By the way, I've restored this test with a minor change. Thanks